lib/fxpay/index.js

```diff
@@ -39,6 +34,7 @@
 
 
   exports.validateAppReceipt = function validateAppReceipt(callback) {
+    settings.initialize();
```

Comment:
I'm on the fence about needing to include this for things that use settings. It would be easy to miss this in future and there's no way I can think of where we can add a safety net. Is there no way we can tie this to configure?